state-chain/pallets/cf-swapping/src/lib.rs

```diff
@@ -587,6 +588,7 @@ pub mod pallet {
 				origin: swap_origin,
 				swap_type: SwapType::Swap(destination_address_internal),
 				broker_commission: None,
+				ingress_fee: None,
```

Comment:
This isn't true, we took a fee when the deposit was made, we just don't pass it through. We should have a test for this.

Reply:
On further reflection - it's not obvious what to do here - it's definitely inconsistent with the standard swap flow. But it's actually possible no event is emitted here at all... which is even more inconsistent?